.ci/windows-steps.yaml

```diff
@@ -0,0 +1,141 @@
+- task: MSBuild@1
+  inputs:
+    solution: 'armadillo-8.400.0/*.sln'
+    msbuildLocationMethod: 'location'
+    msbuildVersion: $(MSBuildVersion)
+    configuration: 'Release'
+    msbuildArchitecture: 'x64'
+    msbuildArguments: /m /p:BuildInParallel=true
+    clean: false
+    maximumCpuCount: false
+  displayName: 'Build armadillo'
+
+# Configure mlpack
+- powershell: |
+    git clone --depth 1 https://github.com/mlpack/mlpack.git
+    mkdir mlpack\build
+    cd mlpack\build
+
+    cmake $(CMakeGenerator) `
+    $(CMakeArgs-mlpack) `
+    -DBLAS_LIBRARIES:FILEPATH=$(Agent.ToolsDirectory)\OpenBLAS.0.2.14.1\lib\native\lib\x64\libopenblas.dll.a `
+    -DLAPACK_LIBRARIES:FILEPATH=$(Agent.ToolsDirectory)\OpenBLAS.0.2.14.1\lib\native\lib\x64\libopenblas.dll.a `
+    -DARMADILLO_INCLUDE_DIR="..\..\armadillo-8.400.0\include" `
+    -DARMADILLO_LIBRARY="..\..\armadillo-8.400.0\Release\armadillo.lib" `
+    -DBOOST_INCLUDEDIR=$(Agent.ToolsDirectory)\boost.1.60.0.0\lib\native\include `
+    -DBOOST_LIBRARYDIR=$(Agent.ToolsDirectory)\boost_libs `
+    -DBUILD_JULIA_BINDINGS=OFF `
+    -DCMAKE_BUILD_TYPE=Release ..
+  displayName: 'Configure mlpack'
+
+# Build mlpack
+- task: MSBuild@1
+  inputs:
+    solution: 'mlpack/build/*.sln'
+    msbuildLocationMethod: 'location'
+    msbuildVersion: $(MSBuildVersion)
+    configuration: 'Release'
+    msbuildArchitecture: 'x64'
+    msbuildArguments: /m /p:BuildInParallel=true
+    maximumCpuCount: false
+    clean: false
+  displayName: 'Build mlpack'
+
+# Configure mlpack
+- powershell: |
+    cp $(Agent.ToolsDirectory)\boost_libs\*.* mlpack\build\
+    cp $(Agent.ToolsDirectory)\OpenBLAS.0.2.14.1\lib\native\lib\x64\*.* mlpack\build\
+    cp $(Agent.ToolsDirectory)\OpenBLAS.0.2.14.1\lib\native\bin\x64\*.* mlpack\build\
+  displayName: 'Configure mlpack'
+
+- powershell: |
+    mkdir build
+    cp $(Agent.ToolsDirectory)\boost_libs\*.* build\
+    cp $(Agent.ToolsDirectory)\OpenBLAS.0.2.14.1\lib\native\lib\x64\*.* build\
+    cp $(Agent.ToolsDirectory)\OpenBLAS.0.2.14.1\lib\native\bin\x64\*.* build\
+    cd build
+    cmake $(CMakeArgs-models) `
+    -DBLAS_LIBRARIES:FILEPATH=$(Agent.ToolsDirectory)\OpenBLAS.0.2.14.1\lib\native\lib\x64\libopenblas.dll.a `
+    -DLAPACK_LIBRARIES:FILEPATH=$(Agent.ToolsDirectory)\OpenBLAS.0.2.14.1\lib\native\lib\x64\libopenblas.dll.a `
+    -DARMADILLO_INCLUDE_DIR="..\armadillo-8.400.0\include" `
+    -DARMADILLO_LIBRARY="..\armadillo-8.400.0\Release\armadillo.lib" `
+    -DMLPACK_INCLUDE_DIR="..\mlpack\build\include" `
+    -DMLPACK_LIBRARY="..\mlpack\build\lib" `
+    -DBOOST_INCLUDEDIR=$(Agent.ToolsDirectory)\boost.1.60.0.0\lib\native\include `
+    -DBOOST_LIBRARYDIR=$(Agent.ToolsDirectory)\boost_libs ..
+    dir
+  displayName: 'Configure models'
+
+# Build models
+- task: MSBuild@1
+  inputs:
+    solution: 'build/*.sln'
+    msbuildLocationMethod: 'location'
+    msbuildVersion: $(MSBuildVersion)
+    configuration: 'Release'
+    msbuildArchitecture: 'x64'
```

Comment:
Let's switch to `x86` and see if that helps.

Reply:
One more thing @zoq we are also getting warning regarding MLPACK_LIBRARY while configuration. Should we change that too? 
I think so It must be `mlpack/build/Release/mlpack.lib`.

Reply:
Yes, good idea.

Reply:
Sure, I'll make the change. Thanks a lot.

Reply:
Hey @kartikdutt18 MLPACK_INCLUDE_DIR was looking good. You just have to change MLPACK_LIBRARY.

Reply:
Hey @Yashwants19, I'll just push a commit. Thanks a lot.

Reply:
Thanks a lot @zoq, @Yashwants19, The build works fine. To fix the architecture issue I passed Win64 flag and that fixed it. For the other error about mlpack was resolved by the above comments and the rest were related to installing some more boost libs.

Thanks a lot!
Regards.

Reply:
Just cleaning commits now. Thanks a lot.